stream/harfanglab-intel/README.md

```diff
@@ -0,0 +1,181 @@
+# OpenCTI Harfanglab Intel Stream Connector
+
+Table of Contents
+
+- [OpenCTI Harfanglab Intel Stream Connector](#opencti-stream-harfanglab-intel)
+    - [Introduction](#introduction)
+    - [Installation](#installation)
+        - [Requirements](#requirements)
+    - [Configuration variables](#configuration-variables)
+        - [OpenCTI environment variables](#opencti-environment-variables)
+        - [Base connector environment variables](#base-connector-environment-variables)
+        - [Connector extra parameters environment variables](#connector-extra-parameters-environment-variables)
+    - [Deployment](#deployment)
+        - [Docker Deployment](#docker-deployment)
+        - [Manual Deployment](#manual-deployment)
+    - [Usage](#usage)
+    - [Debugging](#debugging)
+
+## Introduction
+
+This connector allows organizations to feed the HarfangLab EDR using OpenCTI knowledge.
+
+```mermaid
+graph LR
+    subgraph OpenCTI
+        direction TB
+        subgraph Observations
+            direction TB
+            OpenCTIIndicator[Indicator]
+            OpenCTIObservable[Observable]
+        end
+    end
+
+    subgraph Harfanglab
+        direction TB
+        subgraph ThreatIntelligence
+            direction TB
+            HarfanglabIOCRule[IOCRule]
+            HarfanglabSigmaRule[SigmaRule]
+            HarfanglabYaraFile[YaraFile]
+        end
+    end
+
+%% Relationships between Indicator and Observable
+    OpenCTIIndicator <-.-|based on| OpenCTIObservable
+%% Indicator's observable generates IOCRule
+    OpenCTIObservable ==>|if pattern_type = = 'stix'| HarfanglabIOCRule
+%% Indicator generates SigmaRule
+    OpenCTIIndicator ==>|if pattern_type = = 'sigma'| HarfanglabSigmaRule
+%% Indicator generates YaraFile
+    OpenCTIIndicator ==>|if pattern_type = = 'yara'| HarfanglabYaraFile
+```
+
+OpenCTI data is coming from import connectors. Once this data is ingested in OpenCTI, it is pushed to a Redis event
+stream. This stream is consumed by the HarfangLab connector to insert intel (IOCs, Sigma rules and Yara files) in the
+HarfangLab platform.
+
+```mermaid
+flowchart LR
+    A1[connector X] -->|imports data| B(OpenCTI)
+    A2[connector Y] -->|enriches data| B(OpenCTI)
+    A3[connector Z] -->|imports data| B(OpenCTI)
+    B(OpenCTI) --> C{Streams events and data}
+    C -->|dispatches to| D1[Redis stream]
+    C -->|dispatches to| D2[Redis stream]
+    C -->|dispatches to| D3[Redis stream]
+    D2 -->|consumed by| E(Harfanglab Stream connector)
```

Comment:
Yes, it may be not very clear.
The goal was to show that OpenCTI can push the data to many streams, and Harfanglab Intel will listen to one of them.
If you think it's better to remove the other streams, I'll do it.